jax/_src/numpy/reductions.py

```diff
@@ -433,13 +433,21 @@ def _average(a: ArrayLike, axis: Axis = None, weights: ArrayLike | None = None,
 @implements(np.var, skip_params=['out'])
 def var(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None,
         out: None = None, ddof: int = 0, keepdims: bool = False, *,
-        where: ArrayLike | None = None) -> Array:
-  return _var(a, _ensure_optional_axes(axis), dtype, out, ddof, keepdims,
+        where: ArrayLike | None = None, correction: int | float | None = None) -> Array:
+  if ddof != 0:
+    if correction is not None:
+      raise ValueError(
+        "ddof and correction can't be provided simultaneously."
+      )
+    correction = ddof
+  if correction is None:
+    correction = 0
+  return _var(a, _ensure_optional_axes(axis), dtype, out, correction, keepdims,
```

Comment:
Slghtly simpler:
```python
if correction is None:
  correction = ddof
else if ddof != 0:
  raise ValueError("ddof and correction can't be provided simultaneously.")
```
Same in `std` below.

Reply:
Actually, given the fact that `ddof` may not be static, we should probably do this to avoid concretization errors:
```python
if correction is None:
  correction = ddof
elif not isinstance(ddof, int) or ddof != 0:
  raise ValueError("ddof and correction can't be provided simultaneously.")
```

Reply:
@jakevdp ok, sounds good, I can add this. Is there any way to check and get this concretization errors with `elif ddof != 0` ?

Reply:
`jax.jit(jax.numpy.var)(x, ddof=2)` will result in `ddof` being traced.

Reply:
Thanks, Jake!

However, doing that we'll have slightly different behaviour between jitted and not jitted code:
1) jitted code when we set both to an argument that is 0 is raising ValueError
```python
import jax
import jax.numpy as jnp

@jax.jit
def func(x, a):
    return x - jnp.std(x, ddof=a, correction=a)

x = jnp.arange(10).reshape(2, 5)
out = func(x, a=0)

> ValueError: ddof and correction can't be provided simultaneously.
```
but 2) same code without jitting can run. 

Is it possible to check ddof value when it is `DynamicJaxprTracer` ?

Reply:
No, if it’s a tracer the value is not known until runtime.

Reply:
I think the difference in behavior here is fine – we could make it truly jit-invariant by setting the default to `_NoValue` or something similar, but I don't think that's worth the added complexity here.